env

```diff
@@ -3,13 +3,18 @@ FINALITY='DATA_STATUS_PENDING'
 AR_FINALITY='DATA_STATUS_ACCEPTED'
 STARTING_BLOCK=880279
 AR_STARTING_BLOCK=880279
+AR_ALTCOINS_STARTING_BLOCK=880279
 ID_UPGRADE_A_BLOCK=899534
 MONGO_CONNECTION_STRING='mongodb://mongo:mongo@localhost:27017'
 ETH_CONTRACT='0x049D36570D4e46f48e99674bd3fcc84644DdD6b96F7C741B1562B82f9e004dC7'
+STRK_CONTRACT='0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d'
+USDC_CONTRACT='0x053c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8'
+USDT_CONTRACT='0x068f5c6a61780768455de69077e07e89787839bf8166decfbf92b645209c0fb8'
 IDENTITY_CONTRACT='0x0783a9097b26eae0586373b2ce0ed3529ddc44069d1e0fbc4f66d42b69d6850d'
 REFERRAL_CONTRACT='0x0312a1339da3cdfcae57401705f32f4356e55c898d75df2f0db9b16f3c364ea7'
 NAMING_CONTRACT='0x003bab268e932d2cecd1946f100ae67ce3dff9fd234119ea2f6da57d16d29fce'
 AUTO_RENEW_CONTRACT='0x07879a0d106d8e48751be4c01373e27bee0613afb0e5b8bbc569610fc0514a3a'
+STRK_AUTO_RENEW_CONTRACT= '0x0'
```

Comment:
I suggest to directly load `AUTO_RENEW_ALTCOINS_CONTRACTS` from here via `AUTO_RENEW_ALTCOINS_CONTRACTS_LEN` like we do with CUSTOM_RESOLVER_0, etc